app/controllers/providers/transactions_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Providers
   class TransactionsController < ProviderBaseController
     def show
+      disregarded_state_benefits_list
```

Comment:
Do something similar to what we already do and `redirect_to(problem_index_path) && return unless disregarded_state_benefits_list`